event_project/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module allows you to create events with assigned a project.
```

Comment:
Add at the end:
"If the event date is changed, then all the pending tasks are retro-planned again according the new date."